src/colorizer/colors/color_ramps.ts

```diff
@@ -14,7 +14,7 @@ export type RawColorData = {
    * Most commonly used for glasbey color ramp visualizations, where lots
    * of colors are needed to distinguish between many different values.
    */
-  categorical?: boolean;
+  type?: ColorRampType;
```

Comment:
Added types field so I can have the color ramp slider show the midpoint when a diverging ramp is selected. This may eventually also help support #617.

<img width="317" height="684" alt="Image" src="https://github.com/user-attachments/assets/5360d1de-9667-4679-b63a-0102ce3b7d02" />